enhancements/observability/cluster-observability-operator-logging-stack.md

```diff
@@ -0,0 +1,405 @@
+7. The COO tracks on a high-level when all stack components (i.e. collection, storage, rules) reach the phase `ready`.
+8. The COO creates an `UIPlugin` resource that will create and manage the [logging-view-plugin](logging-view-plugin) plugin.
+9. The COO creates triggers the [korrel8r](korrel8r) reconciliation if and only if a `MonitoringStack` resource or fallback monitoring resources (See [below](#automated-korrel8r-deployment-with-cmo-uwm)) exist.
+
+#### Upgrades Workflow
+
+For the OpenShift Logging administrator using the COO installation under the proposed system:
+
+1. The user edits the `OperatorConfig` resource by selecting a new OpenShift Logging subscription channel (e.g. `stable-5.9`).
+2. The COO propagates the channel change in the `Subscription` for LO.
+3. The COO tracks the newly selected LO `ClusterServiceVersion` for the new channel until it reaches phase `succeeded`.
+4. The COO tracks the storage and rules components until they reach phase `ready`.
+5. The COO propagates the channel change in the `Subscription` for CLO.
+6. The COO tracks the newly selected CLO `ClusterServiceVersion` for the new channel until it reaches phase `succeeded`.
+7. The COO tracks the collection components until they reach phase `ready`.
+
+### API Extensions
+
+#### OperatorConfig
+
+The `OperatorConfig` resource is the entry point to tell COO on which operator installations it should manage as long as one `LoggingStack` is managed too. It is a cluster-scoped resource.
+
+```yaml
+apiVersion: observability.openshift.io/v1alpha1
+kind: OperatorConfig
+metadata:
+  name: coo-config
+spec:
+  subscriptions:
+  - type: logging
+    installPlanApproval: Automatic
+    channel: stable-5.8
+```
+
+#### LoggingStack
+
+The `LoggingStack` is meant to be the minimum viable declaration of the logging stack possible, e.g. user provides refs to external dependencies like S3-compatible storage credentials, Certificate Authority bundles. It is a namespace-scoped resource with the limitation for now that only one can exist in `openshift-logging`.
+
+```yaml
+apiVersion: logging.observability.openshift.io/v1alpha1
+kind: LoggingStack
+metadata:
+  name: sample-logging-stack
+  namespace: openshift-logging
+spec:
+  monitoringSelector:
+    openshift.io/cluster-monitoring: "true"
+  storage:
+    size: 1x.extra-small
+    storageClassName: gp3-csi
+    storage:
+      secret:
+        name: aws-s3-credentials
+        type: s3
+```
+
+The `UIPlugin` resources are discussed in a [separate proposal](observability-ui-proposal) here.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+N/A.
+
+#### Standalone Clusters
+
+This proposal is mainly targeting standalone clusters either on-premises or OpenShift Dedicated. Users upgrading from existing OpenShift Logging installation need to uninstall them first and re-install through the COO by following the above workflow. Doing so will enable to use the correlation component out of the box.
```

Comment:
I presume users can re-use storage in the new deployment, thus preserving their data.  Consider mentioning it here so they know they don't have to "start fresh"